releasenotes.rb

```diff
@@ -96,17 +130,36 @@ def generate
 
       parts = message.split("\n\n", 2)
 
+      merge_commit = commit.parents.size > 1
+      special_deploy_requirements = merge_commit && commit_tags.special_deploy_requirements?(commit.sha)
+
       commits << OpenStruct.new(
         author:  commit.author.name,
         sha:     commit.sha,
         subject: parts[0],
         body:    parts[1],
         pull:    commit.message.scan(find_pr_regex).first,
         issues:  commit.message.scan(find_issues_regex).first,
+        special_deploy_requirements: special_deploy_requirements,
       )
 
     end
     puts ERB.new(@@GENERATE_TEMPLATE, nil, '<>').result binding
+
+    commits_with_special_deploy_requirements = commits.select(&:special_deploy_requirements)
+    unless commits_with_special_deploy_requirements.empty?
+      puts <<~EOF
+      ====================
+
+      WARNING: There are commits with special deploy requirements!
+      EOF
+
+      commits_with_special_deploy_requirements.each do |commit|
+        Array(commit.pull).each do |pull|
+          puts "* #{opts[:github]}/pull/#{pull}"
```

Comment:
That was originally a problem, which is why we now only grab the commits that are merge commits (`parents.size > 1`).